Core/Utilities/MySQL.py

```diff
@@ -1131,160 +1088,159 @@ def buildCondition( self, condDict = None, older = None, newer = None,
     condition = ''
     conjunction = "WHERE"
 
-    if condDict != None:
+    if condDict is not None:
       for aName, attrValue in condDict.iteritems():
-        if isinstance( aName, basestring ):
-          attrName = _quotedList( [aName] )
-        elif isinstance( aName, tuple ):
-          attrName = '('+_quotedList( list( aName ) )+')'
+        if isinstance(aName, basestring):
+          attrName = _quotedList([aName])
+        elif isinstance(aName, tuple):
+          attrName = '(' + _quotedList(list(aName)) + ')'
         if not attrName:
           error = 'Invalid condDict argument'
-          self.log.warn( 'buildCondition:', error )
-          raise Exception( error )
-        if isinstance( attrValue, list ):
-          retDict = self._escapeValues( attrValue )
+          self.log.debug('buildCondition:', error)
```

Comment:
I am OK to change to `debug` when it was `verbose` but these are `warn`. Shouldn't they be?

Reply:
it does not make sense to log it as warning so deep down in the stack. The error is correctly reported to the higher level, and it's up to them to issue the warning